projects/plugins/jetpack-boost/.husky/.gitignore

```diff
@@ -0,0 +1 @@
+_
```

Comment:
No need for `projects/plugins/jetpack-boost/.husky/` either. The monorepo already has hooks set up.

Reply:
Deleted https://github.com/Automattic/jetpack/pull/20512/commits/20530e155c219bc6baf3899c5469f03dabbe4025